### PR TITLE
[sample-ios] Add 2 manual fingerprint test cases for iOS

### PR DESCRIPTION
**Pull request by qiuzhong**
Add 2 manual fingerprint test cases to test if the Crosswalk
fingerprint extension feature works well.

BUG=https://crosswalk-project.org/jira/browse/XWALK-6234

### CODE REVIEW

**Comment by qiuzhong** [comment]
@Honry

**Comment by qiuzhong** [comment]
Currently, in-app-purchase demo project has been moved from ios-extensions-crosswalk to crosswalk-samples, and fingerprint demo project has not been merged yet. This commit assume all the projects are in crosswalk-samples repo.

File: misc/sampleapp-ios-tests/sampleapp/FingerprintDemo_Build.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
```
Comment (Honry) [inline misc/sampleapp-ios-tests/sampleapp/FingerprintDemo_Build.html]: It's 2016 now.

File: misc/sampleapp-ios-tests/sampleapp/FingerprintDemo_Function.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
```
Comment (Honry) [inline misc/sampleapp-ios-tests/sampleapp/FingerprintDemo_Function.html]: Ditto.

**Comment by qiuzhong** [comment]
@Honry Updated

**Comment by Honry** [comment]
LGTM

**Pull request merged**